cluster/nginx.go

```diff
@@ -250,10 +250,25 @@ func UpdateNginxConfiguration(ctx *Context) chan error {
 																		'$status $body_bytes_sent "$http_referer" '
 																		'"$http_user_agent" "$http_x_forwarded_for"';
 				server {
-					#listen 80 default_server;
-					#listen 443 ssl default_server;
-					server_name _ ;
-					return 404;
+					location / {
```

Comment:
what about /api/ ?

Reply:
This is the nginx.conf file of `nginx-resolver`, we don't need to handle `/api/` here, request are going to be forwarded to `http://m3-portal-proxy:80`, that proxy load balancer knows to route traffic between frontend and backend.